src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -279,14 +279,29 @@ public Uri CreateProjectDashboardUrl(ConnectionInformation connectionInformation
 
         #region Helpers
 
-        private static string CreateUrl(string urlBase, string queryFormat, params string[] args)
+        internal /*for testing purposes*/ static string AppendQueryString(string urlBase, string queryFormat, params string[] args)
```

Comment:
Renamed to make it clearer what this is for, especially against the new `CreateRequestUrl` method.